locksmith/README.md

```diff
@@ -0,0 +1,84 @@
+# Locksmith
+
+Locksmith provides the ability to store metadata associated with Locks outside of the Ethereum blockchain, with reasonable permission around writes.
+
+In the current iteration, reads are publically readable.
+
+## Getting Started
+
+Locksmith currently utilizes [Postgres](https://www.postgresql.org/) as its datastore and will require access to a running Instance. 
+
+You can configure your connection details in a few ways, here are the suggested methods:
+
+1. Configure environment variables (Locksmith with recogize these placed in a .env file at the root of the application)
+   * **DB_USERNAME** - Database User
+   * **DB_PASSWORD** - Password of User
+   * **DB_NAME** - Database Name
+   * **DB_HOSTNAME** - Database Host
+
+
+
+
+   __Note__: The following can be used to quickly get started setting up your __local development__ database.
+   ```sql
+   psql -c 'create database {DATABASE NAME};' -U postgres
+   psql -c "create user {DATASBASE USER NAME} with password '{PASSWORD}';" -U postgres
+   ```
+
+2. If your configuration needs are more demanding than this, further configuration can be established via the storage configuration file  `config\config.js`
+
+Once configured, you will be able to start the application. `npm run dev` or `npm start` will suffice depending on your needs for running Locksmith
```

Comment:
```suggestion
Once configured, you will be able to start the application. `npm run dev` or `npm build && npm start`
will suffice depending on your needs for running Locksmith
```

I am not positive my suggested addition of `npm build &&` is a good idea, but we do require building. input from @julien51 on this?

Reply:
`npm build` would be a NOOP in this case, I feel like we should have a pretty strong case to instruct a user to do this as it has the potential to obscure the way to launch the application